test/unit/routes/upsert-idcoms.test.js

```diff
@@ -24,38 +15,23 @@ const mockHandler = {
 
 console.error = jest.fn();
 
-jest.mock("../../../app/config", () => {
-  return {
-    ...jest.requireActual("../../../app/config"),
-    get isDynamicsIntegration() {
-      return mockIsDynamicsIntegration;
-    },
-  };
-});
-
-let mockIsDynamicsIntegration = true;
-
 describe("upsert idcoms", () => {
   afterAll(async () => {
     jest.resetAllMocks();
   });
 
-  afterEach(async () => {
-    mockIsDynamicsIntegration = true;
-  });
-
   test("should not call the upsert when application id is not specified", async () => {
```

Comment:
can you replace all references to "upsert" with something that makes more sense, ie should not call sendParsed when app... since we aren't upserting anymore